tox.ini

```diff
@@ -18,14 +18,15 @@ application-import-names = rbac, api
 [testenv]
 ;passenv = CI TRAVIS TRAVIS_*
 setenv =
-  DATABASE_NAME={env:DATABASE_NAME:rbac_test}
```

Comment:
I think it might be better to keep the original one.
We have some data setup locally for testing in the database postgres.
The rbac_test would be destroyed. If we change it to postgres, I am afraid it would add some data we don't want to our current db postgres and destroy the db postgres after test finished.

Reply:
Good to know! I reverted the database name back to rbac_test.